src/clj/ataru/virkailija/user/organization_client.clj

```diff
@@ -18,8 +18,9 @@
                                  :type :organization})
 
 (defn- group->map [group] {:name (:nimi group)
-                                 :oid (:oid group)
-                                 :type :group})
+                           :oid  (:oid group)
+                           :type :group
+                           :hakukohderyhma? (if (some->> (:ryhmatyypit group) #(contains? % "hakukohde")) true false)})
```

Comment:
Ainakin mockeissa tuon `:ryhmatyypit` -entryn arvo näyttäis olean vektori, jolloin tuo `contains?` ei toimi. Pitäis olla esim. `#(contains? (set %) "hakukohde")`.

Also vähän lyhempi ois `(boolean (some->> (:ryhmatyypit group) #(contains? (set %) "hakukohde")))`